docs/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: Dusk Documentation
+menubar_toc: true
+nav_order: 1
+---
+
+## Introduction
+Welcome.
```

Comment:
Thank you :3

Reply:
Heh, I still need to write this bit .....